tests/select.spec.ts

```diff
@@ -1,20 +1,204 @@
 import { test, expect } from '@playwright/test';
+import { waitForCustomEvent } from './lib/helper.ts';
 
-test('user is able to make a selection with mouse', async ({ page }) => {
-  await page.goto('/test.html');
+test.beforeEach(async ({page}) => {
+  await page.goto('/select-test.html')
+})
 
-  await expect(page.getByLabel(/bobbins/i)).toBeVisible()
+test('Given an options attribute, on select the options should be visible', async ({ page }) => {
+  const select = page.getByTestId('basic-select')
+  await expect(select).toBeVisible()
 
-  const input = page.getByPlaceholder(/select a bobbins/i).last()
+  // don't expect the options to be rendered until a click on input
+  const optionsContainer = select.locator('.options-container').first()
+  await expect(optionsContainer).not.toBeVisible()
 
-  input.focus()
-  input.type('bogs')
+  // click on the input
+  const input = select.locator('input').first()
+  await input.focus()
```

Comment:
should there be separate tests for display on focus vs. click?

Reply:
so click is tested further down when you click on an option

Reply:
we should also test the initial click to display options though

Reply:
added!